packages/AppImage/install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+wget -O "appimagetool" "https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage"
+chmod a+x appimagetool
+mv appimagetool /usr/bin/
```

Comment:
this requires root

Reply:
Yes, is that a problem?. If yes I can change so it won't require root.

Reply:
I'm not sure. Let's check: https://github.com/g1itch/PyBitmessage/commits/ci-wine-further

Reply:
It should work without root: a67ad09

Reply:
OKay, I'll make the changes.